mysql-test/suite/sys_vars/r/use_socket_sharding_basic.result

```diff
@@ -1,13 +1,13 @@
 SELECT @@GLOBAL.use_socket_sharding;
 @@GLOBAL.use_socket_sharding
-0
+1
 0 Expected
```

Comment:
Do we need to update this line in the test file?  Should it be "0 expected"?  And at line 11 and 24.